packages/website-router/worker.ts

```diff
@@ -70,6 +70,27 @@ async function handleEvent(event: FetchEvent, sentry: Toucan): Promise<Response>
     return redirect(sentry, event.request.url, to);
   }
 
+  // Remove all www && https && http from the URL
+  if (
+    event.request.url.startsWith('www') ||
+    event.request.url.startsWith('https://www') ||
+    event.request.url.startsWith('http://www')
+  ) {
+    const to = event.request.url.replace(/^http(s)?(:)?(\/\/)?|(\/\/)?(www\.)?/i, '').split('.')[0];
```

Comment:
This regex might work but it's too complicated for the mission. 
I think if you'll implement `url.host.startsWith('www.')`, you can just do `url.replace('//www.', '//')`